spec/integration/sign_in_with_redirect_spec.rb

```diff
@@ -4,6 +4,9 @@
 
 describe 'Signing in with a redirect parameter' do
```

Comment:
Maybe the commit message here is misleading – perhaps the previous pass on 2.3 was just lucky that the spec ordering didn't cause a problem? I'll note that when I squash.